GsaGHTests/1_BaseParameters/2_Geometry/GsaElement2dTest.cs

```diff
@@ -386,5 +400,52 @@ private GsaElement2d CreateSampleElement2dWithQuad4Type() {
 
       return elem;
     }
+
+    private GsaElement2d CreateLoadPanel() {
+      var pts = new Point3dList {
+       new Point3d(0, 0, 0),
+       new Point3d(2, 0, 0),
+       new Point3d(2, 2, 0),
+       new Point3d(0, 2, 0),
+     };
+
+      pts.Add(pts[0]);
+      var pol = new Polyline(pts);
+      var elem = new GsaElement2d(pol.ToPolylineCurve());
+
+      int initialElementId = 14;
+      int sectionId = 3;
+      var groups = new List<int>();
+      var dummy = new List<bool>();
+      var names = new List<string>();
+      var off = new List<GsaOffset>();
+      elem.Prop2ds = new List<GsaProperty2d>();
+      for (int i = 0; i < elem.ApiElements.Count; i++) {
+        elem.Ids[i] = initialElementId++;
+        groups.Add(22);
+        dummy.Add(true);
+        names.Add("Shahin");
+        elem.Prop2ds.Add(new GsaProperty2d(sectionId) {
+          ApiProp2d = new Prop2D(),
+        });
+      }
+
+      elem.ApiElements.SetMembers(groups);
+      elem.ApiElements.SetMembers(dummy);
+      elem.ApiElements.SetMembers(names);
+      return elem;
+    }
+
+    [Fact]
+    public void GetCenterPointsReturnsValidPointsForLoadPanel() {
+      GsaElement2d ele = CreateLoadPanel();
```

Comment:
where possible do not use appreviations

```
 GsaElement2d **elements** = CreateLoadPanel();
```

Reply:
Tried to implement as much as possible related to this PR